lib/dora-parser/src/ast/mod.rs

```diff
@@ -599,7 +599,7 @@ impl Modifier {
             Modifier::Open => "open",
             Modifier::Override => "override",
             Modifier::Final => "final",
-            Modifier::Internal => "internal",
+            Modifier::Builtin => "native",
```

Comment:
Don't you think we should name this `Modifier::Native` instead? All other modifiers have the same name as the keyword. I think this is a bit confusing when reading other code, e.g. in `parser.rs`.

Reply:
True. I was concerned that `Modifier::Native` being turned into `FctKind::Builtin` and `FctKind::Native` being something completely different would be somewhat confusing. Shall I rename it?

Or maybe we can find a different name for `FctKind::Native` and its "native" functions (that are unrelated to `@native`)?